### ISSUE TITLE
[bug] Caused by dependency project

### ISSUE THREAD

**Issue by AlexKVal** [karma-runner/karma#1372]
Recently I have found a problem with tests https://travis-ci.org/AlexKVal/react-bootstrap/jobs/58184576 for [react-bootstrap](https://github.com/react-bootstrap/react-bootstrap) project.

I have created a pure trash project (with one simple test) to find out where is the problem with tests runned by Karma.

Here is the travis log https://travis-ci.org/AlexKVal/travis-karma for this project.

And here is the problem:
```console
/home/travis/build/AlexKVal/travis-karma/node_modules/karma/node_modules/memoizee/lib/weak.js:58
				clear: d(map.clear.bind(map))
				                  ^
TypeError: Cannot read property 'bind' of undefined
    at /home/travis/build/AlexKVal/travis-karma/node_modules/karma/node_modules/memoizee/lib/weak.js:58:23
```

It seems the problem is with the new 0.20.3 version of https://github.com/medikoo/es6-weak-map.
With previous v0.20.2 version Karma works fine.

The method `clear()` doesn't exist anymore in `es6-weak-map/polyfill.js`.
Here is the commit [50be019c91](https://github.com/medikoo/es6-weak-map/commit/50be019c91) that removes it.

I will write about the problem to @medikoo.

**Comment by medikoo** [comment]
@AlexKVal thanks for pointing this, WeakMap spec have changed sometime ago (`clear` method was removed), and memoizee have not yet been updated to it. I'm on that now

**Comment by AlexKVal** [comment]
Oh.. Ok. You are fast. 
I just created new `Issue` in your project https://github.com/medikoo/es6-weak-map/issues/5.
Sorry about that.

**Comment by AlexKVal** [comment]
Fixed by [es6-weak-map #5](https://github.com/medikoo/es6-weak-map/issues/5#issuecomment-92257357).
